<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeItWidget</class>
 <widget class="QWidget" name="ChangeItWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeItWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblSysName">
     <property name="text">
      <string>Current Profile: None</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="lstProfiles">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>&amp;Create Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRename">
       <property name="text">
        <string>&amp;Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSelectSaveDir">
       <property name="text">
        <string>&amp;Select Game Directory...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReloadProfiles">
       <property name="text">
        <string>Reload &amp;Profiles</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
